jtravis/src/test/java/fr/inria/spirals/jtravis/helpers/BuildHelperTest.java

```diff
@@ -346,4 +349,37 @@ public void testGetFutureBuildOfBranch2() {
         assertEquals("21420", obtained.getNumber());
     }
 
+    @Test
+    public void testGetBuildsFromSlugWithLimitDate() {
+        String slug = "surli/failingProject";
+        Date limitDate = TestUtils.getDate(2017, 9, 14, 0, 0, 1);
+
+        List<Build> builds = BuildHelper.getBuildsFromSlugWithLimitDate(slug, limitDate);
+        List<String> obtainedIds = new ArrayList<String>();
+
+
+        for (Build b:builds)
+        {
+            obtainedIds.add(b.getNumber());
+        }
+            
+
+        List<String> expectedIds = Arrays.asList("373","374","375");
+        List<String> unexpectedIds = Arrays.asList("273","274","275");
+
+        assertNotNull(builds);
+
+        //Vérifie que tout les ids de la liste expectedIds sont présents dans la liste des ids retournés
+        assertTrue(obtainedIds.containsAll(expectedIds));
```

Comment:
you should use preferrably this syntax: https://stackoverflow.com/a/26429379/750142